src/shogun/kernel/CombinedKernel.cpp

```diff
@@ -84,6 +84,9 @@ bool CCombinedKernel::init_with_extracted_subsets(
 	CFeatures* rf = NULL;
 	CKernel* k = NULL;
 
+    auto l_combined = dynamic_cast<CCombinedFeatures*>(l);
```

Comment:
imo this will be a formatting error :(

Reply:
oh that's odd, no clue why my editor suddenly switched to spaces instead of tabs. (